x-pack/test/functional/apps/remote_clusters/index.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { KibanaFunctionalTestDefaultProviders } from '../../../types/providers';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ loadTestFile }: KibanaFunctionalTestDefaultProviders) => {
+  describe('Remote Clusters app', function() {
+    this.tags('ciGroup4');
```

Comment:
Question: is this supposed to be hardcoded? I am not familiar with the CI groups and naming so just curious 😊

Reply:
I have seen them in other test files and followed suit.

Reply:
I 100% support following existing patterns in the codebase but I think we should also strive to understand the _why_ behind these patterns. Would you mind pinging the folks in the operations team to learn more about what this is for and whether `ciGroup4` is the appropriate one for these tests?